tests/sundials/kinsol_06_v3.output

```diff
@@ -0,0 +1,33 @@
+
+DEAL::Computing residual for the 1th time, at u=10.0000
+DEAL::Setting up Jacobian system at u=10.0000
+DEAL::Computing residual for the 2th time, at u=10.0000
+DEAL::Computing residual for the 3th time, at u=-88.0839
+DEAL::Reporting recoverable failure.
+DEAL::Computing residual for the 3th time, at u=-39.0419
+DEAL::Reporting recoverable failure.
+DEAL::Computing residual for the 3th time, at u=-14.5210
+DEAL::Reporting recoverable failure.
+DEAL::Computing residual for the 3th time, at u=-2.26049
+DEAL::Setting up Jacobian system at u=-2.26049
+DEAL::Computing residual for the 4th time, at u=-2.26049
+DEAL::Computing residual for the 5th time, at u=7.84693
+DEAL::Computing residual for the 6th time, at u=7.84693
+DEAL::Computing residual for the 7th time, at u=2.07902
+DEAL::Setting up Jacobian system at u=2.07902
+DEAL::Computing residual for the 8th time, at u=2.07902
+DEAL::Computing residual for the 9th time, at u=-1.23396
+DEAL::Computing residual for the 10th time, at u=-1.23396
+DEAL::Computing residual for the 11th time, at u=6.16274
+DEAL::KINSOL threw an exception with the following message:
+DEAL::
+--------------------------------------------------------
+An error occurred in file <kinsol.cc> in function
+    unsigned int dealii::SUNDIALS::KINSOL<VectorType>::solve(VectorType&) [with VectorType = dealii::Vector<double>]
+The violated condition was: 
+    status >= 0
+Additional information: 
+    One of the SUNDIALS KINSOL internal functions returned a negative
+    error code: -6. Please consult SUNDIALS manual.
```

Comment:
The demangling here might not be portable between different compilers. Could you just do `exception.print_info(deallog.get_file_stream())` instead?

Reply:
We actually have quite a lot of tests that do that, but I made that change anyway.